wpk/generate_wpk_package.sh

```diff
@@ -101,20 +101,30 @@ function main() {
   do
       case "$1" in
       "-t"|"--target-system")
-          if [[ -n "$2" ]] && [[ "$2" == "linux" || "$2" == "windows" ]]; then
-            local TARGET="$2"
-            local HAVE_TARGET=true
-            shift 2
+          if [[ -n "$2" ]]; then
```

Comment:
This change isn't necessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/255/commits/ac22541a388ac6ed3a45b864fe89d04baccd4d20